### ISSUE TITLE
row names in R script crashing

### ISSUE THREAD

**Issue by ebioman** [fritzsedlazeck/SURVIVOR#20]
Hi
The R-script which is supposed to visualize the summary output is not working.

    Error in Math.factor(t[, 1]) : 'log10' not meaningful for factors

Which is not surprising as it takes the first column for the x-axis and transforms it into log10:

    t=read.table(file,header=T)
    plot(log10(t[,1]),t[,2],xlab="log10(length(bp))",ylab="# of SVs",type='l',col=cols[1],main=fie)

But the format of the file has changed (at least that's my guess):

<pre><code>    
pbsv.summary
Len     Del     Dup     Inv     INS     TRA     UNK
0-50bp  0       0       0       0       0       0
50-100bp        1038    0       0       2875    0       0
100-1000bp      1162    0       0       2507    0       0
1000-10000bp    241     0       0       108     0       0
10000+bp        0       0       0       0       0       0
</code></pre>

As it is now a string and no number any-more the match function reports an error

**Comment by fritzsedlazeck** [comment]
Sorry I will look at this. This table has changed heavily and thus the R script is out of date.
I am not sure if you need to run it on this table anyways. 
The other code in the R script is for the summary file across the CHRs, which I also havent used in a while... 

Thanks for reporting this. 
Fritz

**Comment by ebioman** [comment]
Hi
Indeed, it is not a crucial script and one can easily play around a bit to fix it.
Just wanted to provide feedback ;)

**Comment by fritzsedlazeck** [comment]
Yep no fair enough. Thanks for pointing it out. Its not high on the priority list but i will keep this open.
